### PR TITLE
Durus

### PR DESCRIPTION
**Pull request by grindhold**
###### Description of changes

I added the database module `durus` to pythonPackages.

###### Things done

- Built on platform(s)
  - [ x ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/durus/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    owner = "nascheme";
+    repo = "durus";
+    rev = "v${version}";
+    sha256 = "sha256:1kjiyfv5cb36g563bawj9hx563r3shhmcq77flkqb93b80qvkbqv";
+  };
+
+  nativeBuildInputs = [ ];
+  buildInputs = [  ];
+
+  doCheck = false;
```
Comment (sternenseemann) [inline pkgs/development/python-modules/durus/default.nix:17]: Why are you not executing the test suite? If it's not possible in the Nix build sandbox or unfeasible, please add a comment noting why.

Test suites are relatively important for python packages as they are virtually the only sanity checking possible with it lacking type checking etc.

If you can't enable tests, add an `pythonImportsCheck` at least.

File: pkgs/development/python-modules/durus/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  pname = "durus";
+  version = "4.2";
+
+  src = fetchFromGitHub {
```
Comment (sternenseemann) [inline pkgs/development/python-modules/durus/default.nix:7]: In general, I'd prefer to fetch the release tarballs from Pypi via `fetchPypi` unless there's a reason against it.

**Comment by sternenseemann** [comment]
Can you rebase on master?

File: pkgs/development/python-modules/durus/default.nix (hunk 1 above)
Comment (grindhold) [reply pkgs/development/python-modules/durus/default.nix:18]: added a comment explaining why tests are ommited.

File: pkgs/development/python-modules/durus/default.nix (hunk 2 above)
Comment (grindhold) [reply pkgs/development/python-modules/durus/default.nix]: done

**Pull request closed without merging**